packages/api-db-mongodb/src/db/schema.ts

```diff
@@ -178,6 +183,34 @@ export interface DBImage {
   focalPoint?: FocalPoint
 }
 
+export interface DBComment {
+  _id: any
+
+  createdAt: Date
+  modifiedAt: Date
+
+  articleID?: string
+  imageID?: string
+
+  siteID?: string
+  userID: string
+  peerID?: string
+  permalink?: string
+
+  revisions: [DBCommentRevision]
+  parentID?: string
+
+  status: CommentStatus
+  rejectionReason?: CommentRejectionReason
+  authorType: CommentAuthorType
+}
+
+export interface DBCommentRevision {
+  id: string
```

Comment:
Do you need to make `DBCommentRevision`? Can't you use the interface `CommentRevision` from `packages/api`?

Reply:
fixed